operator/api/v1beta1/helmrelease_types.go

```diff
@@ -69,12 +69,12 @@ type HelmReleaseCondition struct {
 // ChartSpec defines the desired Helm chart
 type ChartSpec struct {
 	Repository string `json:"repository"`
-	Path       string `json:"path"`
-	Revision   string `json:"revision"`
+	Name       string `json:"name"`
```

Comment:
Looking at the helm docs, they refer to the chart URL as being made up of a repository and a name, and then you can supply a version as a flag. Just want to mirror this here.